lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -164,43 +217,87 @@
 
 
 .object-type-filter-link {
-    padding: 10px 15px;
     font-size: 1.3em;
+    padding: 18px 15px;
 
     &:global(.active),
     &:hover {
-        text-decoration: none;
-        background-color: $color-bg-gray;
+        background-color: $color-bg-white;
+        border-bottom: 1px solid $color-text-black;
         color: $color-text-black;
-        border-bottom: 2px solid $color-text-black;
+        text-decoration: none;
     }
 }
 
 .sort-dropdown {
-    width: 170px;
-    margin-top: 10px;
+    background-color: $color-bg-gray-blue-light;
+
+    p {
+        margin: 0 0 0 2px;
+    }
+
+    div {
+        background-color: $color-bg-gray-blue-light;
+        border: 0 solid transparent;
+        color: $color-bg-blue-dark;
+        width: 170px;
+    }
+
+    > div {
+        display: flex;
+    }
+}
+
+.ember-power-select-status-icon {
+    right: 10px;
+    top: 12px;
 }
 
 .sidenav-toggle {
     float: left;
 }
 
 .no-results {
-    margin: 10px 20px;
     font-size: 1.2em;
+    margin: 10px 20px;
+}
+
+.left-panel-mobile {
+    padding: 1rem;
+    width: 75vw;
+
+    div:nth-of-type(2) {
```

Comment:
Let's use a local-class for this, as this makes the intended target for this style clearer and also will prevent any unexpected styles if the page layout changes